docs/rules.md

```diff
@@ -559,6 +559,8 @@ A war goal added in this way will result in a status quo war goal on that side b
 
 Any war ending probably requires checking that units aren't stuck in invalid combats, updating blockade status, etc.
 
+A nation being fully annexxed (i.e no longer exists) will automatically white peace all wars it was involved in and remove itself from war participations, it will also end all wars it was the primary attacker or defender.
```

Comment:
"automatically white peace all wars it was involved in" sounds like it would end the wars. But what you mean presumably is that it will end its participation in the wars. Fortunately, this is automatically handled by the fact that deleting the nation also destroys all of its relationships, including war participation